zerver/tests/test_openapi.py

```diff
@@ -276,6 +287,68 @@ def check_for_non_existant_openapi_endpoints(self) -> None:
                 msg += "\n + {}".format(undocumented_path)
             raise AssertionError(msg)
 
+    def _highest_priority_type(self, types: List[type]) -> type:
+        priority = {list: 1, str: 2, int: 3, bool: 4}
+        tyiroirp = {1: list, 2: str, 3: int, 4: bool}
```

Comment:
This sort of clever name is confusing; go with `type_to_priority` and `priority_to_type` for these.

Reply:
```
======================================================================
FAIL: test_openapi_arguments (zerver.tests.test_openapi.OpenAPIArgumentsTest)
This end-to-end API documentation test compares the arguments
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/srv/zulip/zerver/tests/test_openapi.py", line 457, in test_openapi_arguments
    self.check_argument_types(function, openapi_parameters)
  File "/srv/zulip/zerver/tests/test_openapi.py", line 350, in check_argument_types
    raise AssertionError(msg)
AssertionError: There seems to be a mismatch between the types for the request parameters documented in the OpenAPI documentation and the actual request parameters in the function decleration for update_message_flags.
=================================================================
Parameter          OpenAPI Type         Function Decleration Type
=================================================================
op                <class 'int'>         <class 'str'>


----------------------------------------------------------------------
```